### PR TITLE
Feat/panning fixes

### PR DESCRIPTION
**Pull request by M-i-k-e-l**

### CODE REVIEW

File: src/components/panningViews/panDismissibleView.js [hunk 1]
```diff
@@ -193,8 +210,9 @@ class PanDismissibleView extends PureComponent {
   };
 
   onInitAnimationFinished = () => {
-    this.setState({isAnimating: false});
-    this.initPositions();
+    const {forceDismissIsPending} = this.state;
+    const runAfterSetState = forceDismissIsPending ? this.animateDismiss : undefined;
+    this.initPositions({isAnimating: false, forceDismissIsPending: false}, runAfterSetState);
```
Comment (ethanshar) [inline src/components/panningViews/panDismissibleView.js]: do we really need `isAnimating` in state

File: src/components/panningViews/panDismissibleView.js [hunk 2]
```diff
@@ -51,6 +56,7 @@ class PanDismissibleView extends PureComponent {
       bounciness: DEFAULT_BOUNCINESS,
       duration: DEFAULT_DISMISS_ANIMATION_DURATION,
     },
```
[the remark this answers is not in the document]
Comment (M-i-k-e-l) [reply src/components/panningViews/panDismissibleView.js]: Done

File: src/components/panningViews/panDismissibleView.js [hunk 3]
```diff
@@ -193,8 +210,9 @@ class PanDismissibleView extends PureComponent {
   };
 
   onInitAnimationFinished = () => {
```
[the remark this answers is not in the document]
Comment (M-i-k-e-l) [reply src/components/panningViews/panDismissibleView.js]: Done

File: src/components/panningViews/panDismissibleView.js (hunk 1 above)
Comment (M-i-k-e-l) [reply src/components/panningViews/panDismissibleView.js]: We need it in the state (but I removed `shouldDismissAfterReset` from the state)

File: src/interactableComponents/drawer/newDrawer.js [hunk 4]
```diff
@@ -25,7 +25,8 @@ const ITEM_PROP_TYPES = {
 
 /**
  * @description: Drawer Component
```
Comment (ethanshar) [inline src/interactableComponents/drawer/newDrawer.js]: I already fixed this in master, please remove it from the PR

Comment (M-i-k-e-l) [reply src/interactableComponents/drawer/newDrawer.js]: Merged, done

**Pull request merged**